Task.php

```diff
@@ -0,0 +1,70 @@
+<?php
+class Task
+{
+    private $status;
+    private $action;
+    public $next_actions_work = [];
```

Comment:
Согласно PRS именование переменных должно быть в camelCase стиле
```suggestion
    public $nextActionsWork = [];
```